vic/drivers/image/src/vic_force.c

```diff
@@ -192,6 +194,105 @@ vic_force(void)
     // Update the offset counter
     global_param.forceoffset[0] += NF;
 
+    // Initialize the veg_hist structure with the current climatological
+    // vegetation parameters.  This may be overwritten with the historical
+    // forcing time series.
+    for (i = 0; i < local_domain.ncells_active; i++) {
+        for (v = 0; v < options.NVEGTYPES; v++) {
+            vidx = veg_con_map[i].vidx[v];
+            if (vidx != -1) {
+                for (j = 0; j < NF; j++) {
+                    veg_hist[i][vidx].albedo[j] =
+                        veg_con[i][vidx].albedo[dmy[current].month - 1];
+                    veg_hist[i][vidx].LAI[j] =
+                        veg_con[i][vidx].LAI[dmy[current].month - 1];
+                    veg_hist[i][vidx].vegcover[j] =
+                        veg_con[i][vidx].vegcover[dmy[current].month - 1];
+                }
+            }
+        }
+    }
```

Comment:
I don't care about the performance.  I'm mostly concerned with unintended outcomes.  It sounds like this is the right way to go.